zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -2583,18 +2588,23 @@ components:
           format: int64
         state:
           type: string
```

Comment:
🔧 By the way, all of the user task search components don't have descriptions. That makes it harder for users to understand the properties.

Reply:
Sounds great :) 
I have added the more descriptions here https://github.com/camunda/camunda/pull/23598#:~:text=on%20rest%2Dapi-,33b21de,-Merge%20state 

I am going to tag a tech written for review ;).